crates/swc_ecma_transforms_base/tests/ts-resolver/ts_resolver_nested_enum/output.ts

```diff
@@ -1,11 +1,9 @@
 enum Foo__2 {
-    name__0,
-    string__0
+    name__0
```

Comment:
I think this change is acceptable because the input(`ts_resolver_nested_enum`) contains invalid syntax.